index.html

```diff
@@ -24,20 +26,104 @@
+      <p class="short-description w-90">LOCUS is the biggest Technical Fest in Nepal. It has a long history of 20 years.
+        Let's celebrate this March on the name of new innovation!</p>
+      <div class="time w-90">2023.03.01(Wed) ~ 08(Wed)</div>
+      <div class="venue w-90">@ Cricket Ground, Institute of Engineering, Pulchowk</div>
+    </section>
+
+    <section class="department">
+     
+      <h2 class="department-header">Departments</h2>
+      <hr>
+      <div class="wrap-engineering">
+        <section class="engineering">
+          <img src="assets/civil.png" alt="">
+          <h3 class="department-name">Civil Engineering</h3>
+          <p class="description">The design, construction, and maintenance of the built environment, including structures such as buildings, roads, bridges, and water and sewage systems.</p>
+        </section>
+  
+        <section class="engineering">
+          <img src="assets/electrical.png" alt="">
+          <h3 class="department-name">Electrical Engineering</h3>
+          <p class="description">The study and application of electricity, electronics, and electromagnetism, including the design and development of electrical systems and equipment, such as power generation and distribution systems, electrical motors, and lighting systems.</p>
+        </section>
+  
+        <section class="engineering">
+          <img src="assets/electronics.png" alt="">
+          <h3 class="department-name">Electronics Engineering</h3>
+          <p class="description">The design and development of electronic devices and systems, such as computers, communication systems, and control systems.</p>
+        </section>
+  
+        <section class="engineering">
+          <img src="assets/computer.png" alt="">
+          <h3 class="department-name">Computer Engineering</h3>
+          <p class="description">Combines principles from both electrical engineering and computer science to design and develop computer systems and network infrastructure.</p>
+        </section>
+  
+        <section class="engineering">
+          <img src="assets/mechanical.png" alt="">
+          <h3 class="department-name">Mechanical Engineering</h3>
+          <p class="description">The design, development, and operation of machinery and mechanical systems, including power-producing machines such as internal combustion engines and turbines, and power-using machines such as refrigeration and air-conditioning systems.</p>
+        </section>
+      </div>
+  
+      <button class="button-participate">LOCUS Participation 2023</button>
+      <span class="see-more">SEE ALL PROGRAMS</span>
+    </section>
+  
+    <section id="committee" class="committee">
+      <h2 class="committee-header">Committee</h2>
+      <hr>
+      <div id="member-wrap" class="member-wrap"></div>
+      <button class="more">MORE  <span><i class="fa-solid fa-chevron-down"></i></span></button>      
+    </section>
+
```

Comment:
- [ ] [Optional] If you see the background of home page then you will find It has a white background. Please see below and change it.
### Your design
![your](https://user-images.githubusercontent.com/81727988/218167693-d961ffb9-da21-4668-bf31-3e7bb52d145d.png)


### Original design
![original](https://user-images.githubusercontent.com/81727988/218167474-373e70c5-8518-49ff-b2ec-8c049838261c.png)

Reply:
I have used the color swatch given in the design. The original design does not have a #fff background (You can notice, it is slightly different from the nav-bar background color and there is a border #fff in the paragraph section, therefore home page couldn't have background #fff). As you have mentioned this suggestion is optional, I will try making changes in the upcoming days. Thank you.